cf-serverd/server_common.c

```diff
@@ -459,13 +459,6 @@ void CfGetFile(ServerFileGetState *args)
     }
     else
     {
-        int div = 3;
-
-        if (sb.st_size > 10485760L) /* File larger than 10 MB, checks every 64kB */
-        {
-            div = 32;
-        }
```

Comment:
Nice. `stat()` is a syscall, but it has pretty good caching mechanisms on modern systems so it's not that expensive. And old crap will always behave like old crap...

Reply:
Added this note to the commit message. Thanks 😉